com/bl/ttt/main.py

```diff
@@ -1,43 +1,73 @@
-from bl.ttt import ttt_game
-
-# Instantiate the game instance, which gets the initial board setup from the user
-game = ttt_game.Tictactoe()
-# Run the game in a loop
-while True:
-    game.display()
-    current_status = game.status_check()
-    if current_status == 'Game not finished':
-        game.input_move()
-        # Flip players between turns
-        if game.player == "X":
-            game.player = "O"
-        elif game.player == "O":
-            game.player = "X"
+from com.bl.ttt.tictactoe import Tictactoe, TictactoeMove
+
+game_over = False
+player_tries_again = True
+get_next_move = True
+
+
+def process_another_move(game):
+    player_id, cell = input("Please enter player X or O, a space, and then a cell from 0 to 8: ").split(' ')
+    player_id = player_id.upper()
+    cell = int(cell)
+    if player_id is not None and (player_id == 'O' or player_id == 'X'):
+        if cell is not None and 0 <= cell <= 8:
+            move = TictactoeMove(player_id, cell)
```

Comment:
Seems like some of the validation has crept into the orchestrator ?

Reply:
This seemed to be a minimum to reprompt for valid input against the is_valid_move method: should it be removed?

Reply:
That puts validation logic in two places - does that seem right to you ?

Reply:
No, on reflection is does not.